### PR TITLE
Add third party package Startlette-Login

### PR DESCRIPTION
**Pull request by jockerz**
Hi guys

I request a small change on the documentation part.
I want to add a new third party package, that developed by extend the usage of Starlette on handling session based authentication.

- [x] Initially raised as discussion [1639](https://github.com/encode/starlette/discussions/1639)

### CODE REVIEW

File: docs/third-party-packages.md [hunk 1]
```diff
@@ -97,6 +97,15 @@ A plugin for providing an endpoint that exposes [Prometheus](https://prometheus.
 
 A simple tool for integrating Starlette and WTForms. It is modeled on the excellent Flask-WTF library.
 
+### Starlette-Login
+
+<a href="https://github.com/jockerz/Starlette-Login" target="_blank">GitHub</a> |
+<a href="https://starlette-login.readthedocs.io/en/stable/" target="_blank">Documentation</a>
```
Comment (Kludex) [inline docs/third-party-packages.md:103]: Is the link this one or https://starlette-login.readthedocs.io/en/latest/ ?

**Comment by Kludex** [comment]
Thanks for the PR 🙏

**Review by Kludex — APPROVED** [review]

File: docs/third-party-packages.md (hunk 1 above)
Comment (jockerz) [reply docs/third-party-packages.md:103]: Yes

**Comment by jockerz** [comment]
Thank you for your approval. :)

**Comment by Kludex** [comment]
Thanks for the PR! 🙏

**Comment by jockerz** [comment]
> Thanks for the PR! pray

You are welcome

**Pull request merged**